unified-codes/libs/ui/src/lib/components/entity/organisms/EntityBrowser.tsx

```diff
@@ -78,48 +80,57 @@ export const EntityBrowser: EntityBrowser = ({
     onClear && onClear();
   };
 
-  const { page = 1, rowsPerPage = 25 } = variables || {};
+  const { page = 1, rowsPerPage = 10 } = variables || {};
+  const entityTypes = [
+    { name: 'Drugs', active: true },
+    { name: 'Unit of Use', active: false },
+    { name: 'Other', active: false },
+  ];
+
   return (
-    <Grid container direction="column" className={classes?.root}>
-      <Grid item className={classes?.searchBar}>
-        <SearchBar
-          input={input}
-          label="Search description"
-          onChange={onChangeInput}
-          onClear={handleClear}
-          onSearch={onSearch}
-        />
-      </Grid>
-      {entities.totalLength ? (
-        <>
-          <Grid item className={classes?.table}>
-            <EntityTable
-              data={entities.data}
-              {...childProps}
-              onSort={onSort}
-              variables={variables}
-            />
-          </Grid>
-          <Grid item className={classes?.pagination}>
-            {entities.totalLength && (
-              <TablePagination
-                rowsPerPageOptions={[10, 25, 100]}
-                component="div"
-                count={entities.totalLength}
-                rowsPerPage={rowsPerPage}
-                page={page}
-                onChangePage={handleChangePage}
-                onChangeRowsPerPage={handleChangeRowsPerPage}
+    <>
```

Comment:
Oh - I see now why it kind of fits into libs better. I was first thinking it was too application specific to put here, but makes sense given the entity-component subfolder and EntityBrowser setup.